adapters/Vungle/VungleAdapter/Bidding/GADMediationVungleNativeAd.m

```diff
@@ -226,12 +230,16 @@ - (void)willCloseAd {
     // Do nothing. Native ads utilize a different set of callbacks.
 }
 
-- (void)willLeaveApplication {
+- (void)didCloseAd {
     // Do nothing. Native ads utilize a different set of callbacks.
 }
 
-- (void)willShowAd {
+- (void)willLeaveApplication {
     // Do nothing. Native ads utilize a different set of callbacks.
 }
 
+- (nullable NSString *)bidResponse {
```

Comment:
@stanley-vungle We don't support HB Native ad at this point. Will introducing this method give a user to request HB ad?

Reply:
No, it will not give a user the ability to setup hb native ads because it is not available on the dashboard. This value will always be nil and isn't used currently. But I added it in here so when hb comes, I don't need to change it. I can change this to return nil if you think it is better.

To enable hb setup, we need to request it from Google first. Right now, only waterfall is requested.

Reply:
Got it. Thanks for confirming.